room/compiler-processing/src/main/java/androidx/room/compiler/processing/ksp/KspProcessingEnv.kt

```diff
@@ -38,7 +38,8 @@ internal class KspProcessingEnv(
     override val options: Map<String, String>,
     codeGenerator: CodeGenerator,
     logger: KSPLogger,
-    val resolver: Resolver
+    val resolver: Resolver,
+    override val targetLanguage: XProcessingEnv.Language = XProcessingEnv.Language.KOTLIN
```

Comment:
ditto. this should default to java.

though i'm wondering, given that we require these in the builders, can we remove the default here?

Reply:
We could, but I was wondering if this could break client code. Will see if it breaks compilation or tests. If it passes I´ll remove it.

Reply:
[`SyntheticKspProcessor`](https://github.com/androidx/androidx/blob/788e5b8a5af3ca6102837b93ed272c60d4945f27/room/compiler-processing-testing/src/main/java/androidx/room/compiler/processing/SyntheticKspProcessor.kt#L54) seems to make use of that factory. Should I explicitly pass Java to it?

Reply:
yep. at least for now until we add this language parameter to the `runProcessorTest` function.

Reply:
Will take a look into it